commands.js

```diff
@@ -440,6 +440,7 @@ var commands = exports.commands = {
 	roomalias: function (target, room, user) {
 		if (!room.chatRoomData) return this.sendReply("This room isn't designed for aliases.");
 		if (!target) {
+			if (!this.canBroadcast()) return;
```

Comment:
Why would anyone need to broadcast a room alias? This command is for making them.